app/controllers/api/account.php

```diff
@@ -1647,6 +1654,99 @@
         throw new Exception('Session not found', 404);
     });
 
+App::patch('/v1/account/sessions/:sessionId/oauth2-tokens')
```

Comment:
I'm not sure this path makes total sense. I'd love us to come with other REST complaint ideas.

Reply:
Changed to `PATCH /v1/account/sessions/:sessionId`, and we can use same endpoint for session renewing in the future.